website/docs/04 - Model/03 - Reading By Id.md

```diff
@@ -26,10 +26,11 @@ static findById(id: string, options?: ModelFindByIdOptions): Promise<Model | nul
 
 #### Options Object Properties
 
-| Property      | Type       | Description                                                                                                    |
-| ------------- | ---------- | -------------------------------------------------------------------------------------------------------------- |
-| `projection`  | `string[]` | The [projection](./Advanced%20Topics/model_projection) of properties to return from the database               |
-| `userDefined` | `object`   | The [user defined options](./Advanced%20Topics/model_user_defined_options) to pass to the database subroutines |
+| Property               | Type       | Default         | Description                                                                                                    |
+| ---------------------- | ---------- | --------------- | -------------------------------------------------------------------------------------------------------------- |
+| `maxReturnPayloadSize` | `number`   | `1,000,000,000` | The maximum allowed return payload size specified in bytes. A DbServerError will be thrown                     |
```

Comment:
I'm not sure if specifying a default for each method is correct.  If not specified, it falls back to the connection's value.  I'm thinking it should either be omitted or some annotation indicating that it will fall back to the connection's value would be appropriate here.